<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checkout</class>
 <widget class="QWidget" name="Checkout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="PaymentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="PaymentLayout">
     <item>
      <widget class="QLabel" name="subTotal">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Subtotal:  $20.00</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
</string>
       </property>
       <property name="title">
        <string>Pick Payment Method</string>
       </property>
       <widget class="QRadioButton" name="CashButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>201</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Cash on Delievery</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="CreditButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>181</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Credit Card</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="CreditWidget" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>210</y>
     <width>511</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="CreditCardInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>459</width>
      <height>319</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Credit Card Info</string>
    </property>
    <widget class="QLabel" name="CreditNumLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Credit Card Num:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CreditNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CVVLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CVV</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CVV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ExpDateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Expiry Date:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Month">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>82</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Year">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>82</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2031</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2032</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2033</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2034</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="SaveCredit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Credit Card Info</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>790</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 184, 158);</string>
   </property>
   <property name="text">
    <string>Order Confirmation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
